juniper_official/Solutions/Traffic-Blackhole/fabric-stats.rule

```diff
@@ -1,171 +0,0 @@
-/*
-* Rule to get all the Fabric stats (input/output) stats
```

Comment:
Lets not remove this until we are sure that FH is not using this.
Add back this file and add a comment at the top as DEPRECATED

Reply:
Please add DEPRECATED note at the top of the file in the comments